src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +91,21 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return someDate.compareTo(LocalDate.now()) == 0
```

Comment:
let's create a variable and save the result of `LocalDate.now()` then we can use this variable at this method